.github/workflows/test.yml

```
@@ -254,12 +297,14 @@ jobs:
         assert callable(rookie_cookies.any_browser)"
 
     - name: Get Playwright version
+      if: matrix.canonical
       id: playwright-version
       shell: bash
       run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:80: Double quote to prevent globbing and word splitting

(shellcheck)